src/parquet/reader.cc

```diff
@@ -288,17 +288,18 @@ void ParquetFileReader::DebugPrint(std::ostream& stream, bool print_values) {
     }
     stream << "\n";
 
-    bool hasRow;
-    do {
-      hasRow = false;
+    for (size_t r = group_reader->column_metadata(0)->num_values; r >0; r--) {
```

Comment:
This is a cosmetic change, to prevent DebugPrint from printing an empty row at the end of the output.